test/service/storage/elasticsearch.test.js

```diff
@@ -2060,6 +2060,7 @@ describe('Test: ElasticSearch service', () => {
         mappings = { properties: { city: { type: 'keyword' } } };
 
       elasticsearch.hasCollection = sinon.stub().resolves(true);
+      elasticsearch.deleteCollection = sinon.stub().resolves();
       elasticsearch.updateCollection = sinon.stub().resolves({});
```

Comment:
(nitpicking) Prefer the `sinon.stub(<object>, <property>)` form, as it allows to detect obsolete tests, as sinon will complain if the property to stub does not exist:

```suggestion
      sinon.stub(elasticsearch, 'hasCollection').resolves(true);
      sinon.stub(elasticsearch, 'deleteCollection').resolves();
      sinon.stub(elasticsearch, 'updateCollection').resolves({});
```

Reply:
In the case of `sinon.stub(elasticsearch, 'hasCollection').resolves(true);`, the syntax you suggest won't work since Mocha complains that it's already stubbed.
Using `elasticsearch.hasCollection = sinon.stub().resolves(true);` overwrite it